cls/SourceControl/Git/Utils.cls

```diff
@@ -2186,12 +2261,6 @@ ClassMethod GitStatus(ByRef files, IncludeAllFiles = 0)
     }
 }
 
-ClassMethod EmptyInitialCommit()
```

Comment:
Why was this removed?

Reply:
This was removed because, instead of doing an empty initial commit, we now export the git-source-control.json file (soon to be renamed) and include that in the initial commit.